tests/cephadm/test_iscsi_multiple_removal_deployment.py

```diff
@@ -0,0 +1,89 @@
+import json
+
+from ceph.ceph_admin.common import fetch_method
+from ceph.ceph_admin.helper import get_cluster_state
+from ceph.ceph_admin.iscsi import ISCSI
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class IncorrectIscsiServiceStatus(Exception):
+    pass
+
+
+def _verify_service(node, should_be_present=True):
+    """
+    Verifies iscsi service is deployed or removed
+    :param node: Mon node
+    :param should_be_present: True if service deployed else False
+    :return: (bool)
+    """
+    # Get iscsi serivice status
+    orch_ls = "ceph orch ls --service_type iscsi -f json-pretty"
+    cmd = f"cephadm shell {orch_ls}"
```

Comment:
instead use this module - https://github.com/red-hat-storage/cephci/blob/master/ceph/ceph_admin/shell.py

Reply:
@vamahaja just a thought, since this is a single operation, should we consider importing the ShellMixin class, which inturn has multiple other imports. 
Proceeded with this approach as it was simple and one time use

Reply:
@pranavprakash20 instead add new functionality for orch to [1] and use it here.
[1] https://github.com/red-hat-storage/cephci/tree/master/cli